samples/tuples/bad_type_destruct.error

```diff
@@ -0,0 +1 @@
+cannot destruct i64 into tuple
```

Comment:
`destruct` feels a bit weird, how do you feel about using `unpack` in the scope of this PR?

Reply:
I agree that is better, will change.